setup-conda/script.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Change to the direction containing the action's code
+pushd $GITHUB_ACTION_PATH
+
+# Create a temporary directory
+mkdir -p conda
+BASE=$(realpath conda)
+
+# GAMS source URL fragment, and path fragment for extracted files
+if [$INSTALLER] == 'anaconda'; then
+  INSTALLER_TYPE == 'Anaconda'
+  case $RUNNER_OS in
+    Linux)
+      PATH_ENDING = conda.sh
+      CACHE_PATH="$GITHUB_ACTION_PATH/$PATH_ENDING"
+      CONDA_OS=Linux
+      FRAGMENT=${INSTALLER_TYPE}3-${VERSION_ANACONDA}-${CONDA_OS}-x86_64.sh
+      ;;
+    macOS)
+      PATH_ENDING = conda.pkg
+      CACHE_PATH="$GITHUB_ACTION_PATH/$PATH_ENDING"
+      CONDA_OS=MacOSX
+      FRAGMENT=${INSTALLER_TYPE}3-${VERSION_ANACONDA}-${CONDA_OS}-x86_64.pkg
+      ;;
+    Windows)
+      PATH_ENDING = conda.exe
+      CACHE_PATH="$GHA_PATH\\$PATH_ENDING"
+      CONDA_OS=Windows
+      FRAGMENT=${INSTALLER_TYPE}3-${VERSION_ANACONDA}-${CONDA_OS}-x86_64.exe
```

Comment:
Since this line is the same in all 3 cases, move it out of the `case`/`esac` statement.

Reply:
I would suggest to replace `PATH_ENDING` with a variable `EXT=exe` etc, and then `CACHE_PATH="$GHA_PATH\\conda.$EXT"` etc.

Reply:
Yes, sounds good!